From c180397f288ed1a09c32689e6ce0a20fb59e6dc1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Radek=20Barto=C5=88?= <radek.barton@microsoft.com>
Date: Tue, 20 May 2025 09:02:55 +0200
Subject: [PATCH 05/11] Allow build of `libsim` for `aarch64-w64-mingw32` and
 `aarch64-pc-cygwin` (#8)

(cherry picked from commit 4c7df05d93ba1af0bde1a74eab2ab0875a8ab835)
---
 gdb/configure.tgt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gdb/configure.tgt b/gdb/configure.tgt
index b9e0f80ce6a..b1132efe90c 100644
--- a/gdb/configure.tgt
+++ b/gdb/configure.tgt
@@ -138,6 +138,10 @@ aarch64*-*-elf | aarch64*-*-rtems*)
 	gdb_target_obs="aarch64-newlib-tdep.o"
 	;;
 
+aarch64-*-mingw* | aarch64-*-cygwin*)
+	gdb_target_obs="windows-tdep.o"
+	;;
+
 aarch64*-*-freebsd*)
 	# Target: FreeBSD/aarch64
 	gdb_target_obs="aarch64-fbsd-tdep.o"
-- 
2.51.0.windows.1

